source/css/db-ui-core.general.scss

```diff
@@ -12,3 +12,7 @@ select,
 textarea {
 	@extend %general-fonts;
 }
+
+body {
+	@extend %db-ui-normal;
```

Comment:
I can't find this declaration rendered within the `db-ui-core.general.css` file on localhost – what would it supposed to include ?

Reply:
db-ui-core.general.scss uses this import: 
`@import "db-ui-core.variables";`
db-ui-core.variables imports this one: 
@import "@db-ui/base/build/scss/db-ui-base";

so this placeholder should be accessible? Maybe we should discuss this file structure in  source/css/** ;)